### PR TITLE
update content for Contributing.md

### PR DESCRIPTION
**Pull request by rdeutz**
Pull Request for Issue #43442

### Summary of Changes
Updated the information.

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -7,19 +7,22 @@ You are welcome to submit a contribution for review and possible inclusion in th
...
+   If it is a PR, include what the issue is, what the PR is addressing, testing instructions and environmental information (PHP version, database driver and version, and other data you can retrieve from your site's system information view) in case the issue is specific to certain environments. If additional information is needed, please be prepared to provide it as our community members review your submission.
 
 4) Report security issues to the Joomla! Security Strike Team (JSST) at security@joomla.org or use the [JSST contact form](https://developer.joomla.org/contact-security-team.html). Please do not use the public tracker for security issues.
 
+5) If you have a feature request open a discussion to define the feature request and discuss possible solutions. Discussions can be coverted into issues when the request is so clear that a developer can start codig on the feature. In this process you will get feedback from the maintainers, if the feature is something for the Joomla core distribution.
```
Comment (brianteeman) [inline .github/CONTRIBUTING.md]:
```suggestion
5) If you have a feature request, then please open a discussion to define the feature request and discuss possible solutions. Discussions can be converted into issues when the request is defined sufficiently that a developer can start coding the feature. In this process you will get feedback from the maintainers, if the feature is something for the Joomla core distribution or not.
```

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -7,19 +7,22 @@ You are welcome to submit a contribution for review and possible inclusion in th
...
+
 Please be patient as not all items will be tested immediately (remember, all bug testing for the Joomla! CMS is done by volunteers) and be receptive to feedback about your code.
 
 #### Branches
-PRs should usually be made to the `4.3-dev` branch as this contains the most recent version of the code.
-There are other branches available which serve specific purposes.
+As long as 4.4 is in maintaince mode, what means that we still fixing bugs, PRs should be made to the `4.4-dev` branch. Merged bugfixes will be upmerged into the current 5.x branch. If a bug is only in the 5.x series the PR should be made to the current 5.x branch (currently 5.1).
```
Comment (brianteeman) [inline .github/CONTRIBUTING.md]:
```suggestion
While 4.4 is in maintenance mode, ie we are still fixing bugs, PRs should be made to the `4.4-dev` branch. Merged bugfixes will be upmerged into the current 5.x branch. If a bug is only in the 5.x series the PR should be made to the current 5.x branch (currently 5.1).
```

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -7,19 +7,22 @@ You are welcome to submit a contribution for review and possible inclusion in th
...
+
 
 | Branch | Purpose |
 | ------ | ------- |
-| 3.10-dev | Branch for the Joomla 3.x series. The 3.10 series release will now only include security patches. |
-| 4.3-dev | Branch for the current minor Joomla version.|
-| 4.4-dev | Branch for the next minor Joomla version. Commits to 4.3-dev will be applied to this branch as well. |
-| 5.0-dev | Branch for the next major Joomla version. New features go into this branch. |
+| 4.4-dev | Branch for the current 4.x Joomla version. Currently in maintaince mode |
```
Comment (brianteeman) [inline .github/CONTRIBUTING.md]:
```suggestion
| 4.4-dev | Branch for the current 4.x Joomla version. Currently in maintenance mode |
```

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (muhme) [reply .github/CONTRIBUTING.md]: typo 5) ... Discussions can be coverted -> converted

**Comment by rdeutz** [comment]
Thanks for the corrections @brianteeman @muhme

**Pull request merged**